app/controllers/searches_controller.rb

```diff
@@ -177,7 +177,7 @@ def common_followers
   def replying
     @user_items = build_user_items(@searched_tw_user.replying) # call users
     @graph = @searched_tw_user.replying_graph
-    @tweet_text = view_context.close_friends_text(@user_items.slice(0, 3).map{|i| i[:target] }, @searched_tw_user)
+    @tweet_text = view_context.close_friends_text(@user_items.slice(0, 3).map { |i| i[:target] }, @searched_tw_user)
```

Comment:
__[rubocop]__
* Line is too long. [81/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)